src/QueryBuilder.php

```diff
@@ -201,6 +201,24 @@ public function whereNestedDoc($column, $query, $boolean = 'and'): self
         return $this;
     }
 
+    /**
+     * Add a 'must not' statement to the query.
+     *
+     * @param  \Illuminate\Database\Query\Builder|static $query
+     * @param  string  $boolean
+     * @return self
+     */
+    public function whereNot($query, $boolean = 'and'): self
+    {
+        $type = 'Not';
```

Comment:
You know I've always wondered if these should be constants, and why in the framework they're not? 🤷🏼‍♂️

Reply:
Yeah that's a really good point - it would feel quite nice to add those as constant in our implementation.